webapp/src/main/resources/public/js/components/workbench/TranslationHistoryModal.js

```diff
@@ -0,0 +1,122 @@
+import PropTypes from 'prop-types';
+import React from "react";
+import {FormattedMessage, injectIntl} from "react-intl";
+import {Button, Modal} from "react-bootstrap";
+import {Table} from "react-bootstrap";
+import StatusGlyph from '../widgets/StatusGlyph';
+import TextUnitSDK from "../../sdk/TextUnit";
+
+class translationHistoryModal extends React.Component {
+    static propTypes() {
+        return {
+            "show": PropTypes.bool.isRequired,
+            "textUnit": PropTypes.object.isRequired,
+            "translationHistory": PropTypes.object.isRequired
+        };
+    }
+
+    renderHistoryItem = (item) => {
+        return item ?
+            (
+                <tr>
+                    <td>{item.createdByUser.username}</td>
+                    <td>{item.content}</td>
+                    <td>{this.convertDateTime(item.createdDate)}</td>
+                    <td><StatusGlyph status={item.status} onClick={() => ""}/></td>
+                </tr>
+            ) :
+            "";
+    };
+
+    /**
+     * @returns {*} Generated content for the git blame information section
+     */
+    rendertranslationHistory = () => {
+        const { translationHistory, textUnit, intl } = this.props;
+
+        return (
+            <Table striped bordered hover responsive="sm">
+                <thead>
```

Comment:
the style looks different from other tables. Can we keep similar CSS as the repository/drop pages?